state-chain/runtime/src/chainflip/ethereum_sc_calls.rs

```diff
@@ -13,10 +14,15 @@ pub enum DelegationApi {
 		operator: <Runtime as frame_system::Config>::AccountId, /* Operator the amount to
 		                                                         * delegate to */
 	},
-	Undelegate {},
+	Undelegate,
 	SetMaxBid {
 		maybe_max_bid: Option<<Runtime as cf_traits::Chainflip>::Amount>,
 	},
+	Redeem {
+		amount: RedemptionAmount<<Runtime as cf_traits::Chainflip>::Amount>,
+		address: EthereumAddress,
+		executor: Option<EthereumAddress>,
```

Comment:
I'm not sure we need to expose all of this externally: 

- amount should always be MAX
- address is always the caller (possibly debateable but I don't see why)
- executor is always `None`